### ISSUE TITLE
DotVVM for Visual Studio: The 'DotvvmPackage' package did not load correctly

### ISSUE THREAD

**Issue by OronDF343** [riganti/dotvvm#1297]
Every time I start Visual Studio 2022, I get this message:
![image](https://user-images.githubusercontent.com/5132940/154318555-eeb69a16-0af0-4b33-ad34-e4c54f7af4cb.png)

The file mentioned contains the following error:
```
SetSite failed for package [DotvvmPackage]
Source: 'System.ComponentModel.Composition'
Description: The composition produced multiple composition errors, with 2 root causes. The root causes are provided below. Review the CompositionException.Errors property for more detailed information.
 1) No exports were found that match the constraint: ContractName DotVVM.Integration.VisualStudio.Dialogs.DialogFactory RequiredTypeIdentity DotVVM.Integration.VisualStudio.Dialogs.DialogFactory
    Resulting in: Cannot set import 'DotVVM.Integration.VisualStudio.VsPackages.DotvvmPackageDependencies.DialogFactory (ContractName="DotVVM.Integration.VisualStudio.Dialogs.DialogFactory")' on part 'DotVVM.Integration.VisualStudio.VsPackages.DotvvmPackageDependencies'.
    Element: DotVVM.Integration.VisualStudio.VsPackages.DotvvmPackageDependencies.DialogFactory (ContractName="DotVVM.Integration.VisualStudio.Dialogs.DialogFactory") --> DotVVM.Integration.VisualStudio.VsPackages.DotvvmPackageDependencies
 2) No exports were found that match the constraint: ContractName DotVVM.Integration.VisualStudio.Common.Licensing.ILoginProvider RequiredTypeIdentity DotVVM.Integration.VisualStudio.Common.Licensing.ILoginProvider
    Resulting in: Cannot set import 'DotVVM.Integration.VisualStudio.VsPackages.DotvvmPackageDependencies.LoginProvider (ContractName="DotVVM.Integration.VisualStudio.Common.Licensing.ILoginProvider")' on part 'DotVVM.Integration.VisualStudio.VsPackages.DotvvmPackageDependencies'.
    Element: DotVVM.Integration.VisualStudio.VsPackages.DotvvmPackageDependencies.LoginProvider (ContractName="DotVVM.Integration.VisualStudio.Common.Licensing.ILoginProvider") --> DotVVM.Integration.VisualStudio.VsPackages.DotvvmPackageDependencies
```

**Comment by tomasherceg** [comment]
We are working on a fix - VS 17.1 changed some APIs we are calling.

**Comment by tomasherceg** [comment]
We've just published the new version 4.0.537.0 of the extension - can you please try it? 
It should work with VS 17.1.

**Comment by OronDF343** [comment]
Thanks, the error no longer appears
